src/App.js

```diff
@@ -1,43 +1,42 @@
 import React from "react";
 import "./App.css";
-import { makeShuffledDeck } from "./utils.js";
+import StartMenu from "./startMenu";
+import Game from "./game"
+
 
 class App extends React.Component {
   constructor(props) {
     // Always call super with props in constructor to initialise parent class
     super(props);
     this.state = {
-      // Set default value of card deck to new shuffled deck
-      cardDeck: makeShuffledDeck(),
       // currCards holds the cards from the current round
       currCards: [],
+      gameMode: "start",  //
+      numberOfGames: 0,  
     };
   }
 
-  dealCards = () => {
-    this.setState((state) => ({
-      // Remove last 2 cards from cardDeck
-      cardDeck: state.cardDeck.slice(0, -2),
-      // Deal last 2 cards to currCards
-      currCards: state.cardDeck.slice(-2),
-    }));
-  };
+  receiveDataFromSMtoApp = (childData) => {
```

Comment:
I think I would call this initGame or something along these lines instead

Reply:
oh yeah agree, I just name it like that for the time being to understand how to write a callback function to pass data from child to parent